Sources/CGFloatExtensions.swift

```diff
@@ -47,5 +47,14 @@ extension CGFloat {
     public static func radiansToDegrees(_ angleInDegrees: CGFloat) -> CGFloat {
         return (180.0 * angleInDegrees) / .pi
     }
-
+    
+    /// EZSE: Returns a random floating point number between 0.0 and 1.0, inclusive.
+    public static func random() -> CGFloat {
+        return CGFloat(Float(arc4random()) / 0xFFFFFFFF)
+    }
+        
+    /// EZSE: Returns a random floating point number in the range min...max, inclusive.
+    public static func random(range: ClosedRange<CGFloat>) -> CGFloat {
```

Comment:
random(in range:ClosedRange<CGFloat>) 
random(in range:Range<CGFloat>) 

what do you think?